### PR TITLE
Lupe

### PR DESCRIPTION
**Pull request by lupeMorales**
add tarjet to links, and fix image preview

### CODE REVIEW

File: src/html/partials/cards.html [hunk 1]
```diff
@@ -23,17 +23,17 @@
...
                 </li>
                 <li>
-                    <a class="icon-email icon-border js-preview-email" href="#">
+                    <a class="icon-email icon-border js-preview-email" href="#" target="_blank">
                         <i class="fa-solid fa-envelope fa-xl"></i>
                     </a>
                 </li>
                 <li>
-                    <a class="icon-linkedin icon-border js-preview-link" href="#">
+                    <a class="icon-linkedin icon-border js-preview-link" href="#" target="_blank">
```
Comment (saramonzon22) [inline src/html/partials/cards.html:31]: Perfecto!! Que buena idea Lupe

**Comment by vvarona** [comment]
Very good, little friend 😘

**Review by Tatiana-Na — COMMENTED** [review]
!Very good job!

**Review by vvarona — APPROVED** [review]

**Review by saramonzon22 — APPROVED** [review]
I like a lot your way to solving it!

**Pull request merged**